ossfuzz/fuzz_data_producer.h

```diff
@@ -17,14 +17,14 @@ FUZZ_dataProducer_t *FUZZ_dataProducer_create(const uint8_t *data, size_t size);
 void FUZZ_dataProducer_free(FUZZ_dataProducer_t *producer);
 
 /* Returns a seed value for the function after this one to consume */
-uint32_t FUZZ_dataProducer_uint32_seed(FUZZ_dataProducer_t *producer, uint32_t min,
+uint32_t FUZZ_dataProducer_uint32(FUZZ_dataProducer_t *producer, uint32_t min,
```

Comment:
I don't get it,
if all you want is to extract a `uint32` to consume it as a `seed` later on,
why are there `min` and `max` bounds part of the interface ?
This seems the exact role of `FUZZ_dataProducer_range32()`.